vg/draw/canvas.go

```diff
@@ -32,6 +32,23 @@ type TextStyle struct {
 
 	// Font is the font description.
 	Font vg.Font
+
+	// Rotation is the text rotation in radians, counterclockwise from
+	// the default drawing direction.
+	Rotation float64
+
+	// XAlign and YAlign specify the alignment of the text.
+	// They are multiplied by the width
+	// and height of each label respectively and the
+	// added to the final location.  E.g., XAlign=-0.5
```

Comment:
I prefer your `(XAlign, YAlign) = (-0.5, -0.5)`. But I also think that this can be improved by changing it to a table of useful examples:
```
// For example:
// 	XAlign	YAlign	text location
// 	-0.5		-0.5		vertically and horizontally centered
// 	-0.5		0		below and horizontally centered
// 	-1		-0.5		vertically centered and right aligned
```
(Please double check that these are right).

Also, if we add functions to set these, maybe mention those functions as a convenient way to set some useful alignments.